syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "Enums.proto";
import public "Credential.proto";

message CredentialSet {
    optional uint32 version = 1;
    optional string nymID = 2;
    optional string masterID = 3;
    optional CredentialSetMode mode = 4;
    optional uint32 index = 5;
    optional Credential masterCredential = 6;
    repeated string activeChildIDs = 7;
    repeated Credential activeChildren = 8;
    repeated string revokedChildIDs = 9;
    repeated Credential revokedChildren = 10;
}
